src/frontends/paddle/src/decoder_proto.cpp

```diff
@@ -161,26 +242,88 @@ ov::element::Type DecoderProto::get_out_port_type(const std::string& port_name)
 }
 
 std::string DecoderProto::get_op_type() const {
-    return get_place()->get_desc().type();
+    return m_op_desc.type();
 }
 
 std::vector<proto::OpDesc_Attr> DecoderProto::decode_attribute_helper(const std::string& name) const {
     std::vector<proto::OpDesc_Attr> attrs;
-    for (const auto& attr : get_place()->get_desc().attrs()) {
+    for (const auto& attr : m_op_desc.attrs()) {
         if (attr.name() == name)
             attrs.push_back(attr);
     }
     FRONT_END_GENERAL_CHECK(attrs.size() <= 1,
                             "An error occurred while parsing the ",
                             name,
                             " attribute of ",
-                            get_place()->get_desc().type(),
+                            m_op_desc.type(),
                             "node. Unsupported number of attributes. Current number: ",
                             attrs.size(),
                             " Expected number: 0 or 1");
     return attrs;
 }
 
+///////////////////////////////////////////////////////////////////////////////////////////////////
+VarDecoderProto::VarDecoderProto(const ::paddle::framework::proto::VarDesc& var_desc)
+            : m_var_desc(var_desc) {
+            OPENVINO_DEBUG << "Paddle VarDesc " << m_var_desc.name() << " has VarType "
+                            << ::paddle::framework::proto::VarType_Type_Name(m_var_desc.type().type()) << std::endl;
+}
+
+ov::element::Type VarDecoderProto::get_data_type() const {
+    const auto& var_type = m_var_desc.type();
+    if (var_type.has_lod_tensor()) { // LoDTensorDescs
```

Comment:
Please also verify that `TYPE_MAP` contains data type.

Reply:
fixed.